tests/editor/utils/attachtoform.js

```diff
@@ -43,7 +43,9 @@ describe( 'attachToForm()', () => {
 		submitStub.restore();
 		form.remove();
 
-		return editor.destroy();
+		if ( editor ) {
```

Comment:
I'm sort of confused here. First, [you make sure it's OK to destroy the editor twice](https://github.com/ckeditor/ckeditor5-core/pull/210/files#diff-fbd6c571314d640867976e2efb7b4f23R201), then you're fixing tests (seeming unrelated to the PR) to make sure the editor is not destroyed twice. Am I missing something?

Reply:
This is a relict of changes to the code that happened in two steps.

First I did that change in the test here. Then I discovered that I would have to change dozens of tests in multiple repos. So, I gave up and instead of fixing all the places where editor is destroyed twice, I made sure it doesn't crash.

So, this change is not needed. I can revert it, if you insist.

Reply:
> So, this change is not needed. I can revert it, if you insist.

Yes, please. Let's keep this PR clean.